build-packages/scandipwa-template-plugin-system/error-handlers/possible-plugin-error.js

```diff
@@ -0,0 +1,35 @@
+/* eslint-disable consistent-return */
+const logger = require('@scandipwa/scandipwa-dev-utils/logger');
+
+/**
+ * This decorator function handles errors if any are thrown from the subject (cb).
+ * The further execution is halted on error. The user is notified.
+ *
+ * @param {function} cb callback to attempt to execute
+ * @param {string} name processable plugin's name
+ * @returns {any} the result of the callback
+ */
+module.exports = (name, cb) => {
+    if (!name) {
```

Comment:
You have a throw and console log in a single file, why is it so? Can you please add a comment in code explaing why?

Reply:
Added